octoprint_octolapse/snapshot.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, settings, dataDirectory, printStartTime, printEndTime=None):
         self.PrintStartTime = printStartTime
         self.PrintEndTime = printEndTime
         self.DataDirectory = dataDirectory
+        self._debug = self.Settings.CurrentDebugProfile()
 
```

Comment:
I don't want the current debug profile stored so that we can change settings on the fly.  Just use self.Settings.CurrentDebugProfile() instead of self._debug.CurrentDebugProfile.  I saw self_debug in the old version of clean snapshots in snapshot.py, and that was a mistake.  However, some changes I merged in already fixed that!